src/dashboard/src/components/filesystem_ajax/views.py

```diff
@@ -246,7 +246,9 @@ def start_transfer(transfer_name, transfer_type, accession, paths, row_ids):
 
         if helpers.file_is_an_archive(path):
             transfer_dir = temp_dir
-            filepath = os.path.join(temp_dir, os.path.basename(path))
+            p = path.split(':',1)[1]
+            logging.exception('splitting path ' + p)
```

Comment:
I think you should use a `logger` here not `logging`, also should probably use the `info` method instead of the `exception` one. Is there a reason you're using the latter?